File: Python_Programs/ML Assignment/Assignment 2/armstrong.py
#Program For Checking the Number is Armstrong or not
import math
num_str=input("\nEnter the Number : ")
tmp=num=int(num_str)
sum=0
while tmp!=0:
    sum=sum+math.pow(tmp%10,len(num_str))
    tmp//=10
if(num==sum):
    print("\n{} is Armstrong Number .".format(num))
else:
    print("\n{} is Not Armstrong Number !".format(num))
print("\n")

File: Java Programmes/College Assignment/Assignment 1/Armstrong.java
import java.util.*;
class Armstrong 
{
    public static void main(String args[])
    {
        int num,dgt=0,tmp,sum=0;
        System.out.print("Enter the Number : ");
        Scanner in = new Scanner(System.in);
        num=in.nextInt();
        tmp=num;
        while(tmp>0)
        {
            tmp=tmp/10;
            dgt++;
        }
        tmp=num;
        while(tmp>0)
        {
            sum=sum + (int)Math.pow(tmp%10, dgt);
            tmp=tmp/10;
        }
        if(sum==num)
             System.out.println(num+" is Armstrong Number.");
        else
            System.out.println(num+" is not an Armstrong Number .");
    }
}
